momentum_scanner_intraday.py

```diff
@@ -102,6 +110,13 @@ def find_rets_24h(vols):
     rets24h.index.name = "Token name"
     return rets24h
 
+def add_volume_marketcap(df):
+    for i in df.index:
+        vol_30, mc_30 = token_volume_marketcap(i)
+        df.loc['30_day_mean_volume'] = vol_30
+        df.loc['30_day_mean_marketcap'] = mc_30
```

Comment:
These two lines repeatedly overwrite the same locations?

Not going to ask for an overall difference, but I think most of these `add_*` functions could return a dataframe which could be joined on later, and it would be pretty easy to not drop n^2 bombs in our dataframe handling. This starts failing _hard_ at moderate size dataframes. Anyway.

Reply:
That's actually a bug, good catch. Fixed.